gammapy/modeling/models/spatial.py

```diff
@@ -521,3 +598,13 @@ def to_dict(self):
         data["filename"] = self.filename
         data["normalize"] = self.normalize
         return data
+
+    @property
+    def to_region(self):
+        """Return model as a `~regions.RectangleSkyRegion`."""
+        return RectangleSkyRegion(
+            center=self.position,
+            width=self.map.geom.width[0][0],
+            height=self.map.geom.width[1][0],
+            angle=0 * u.deg,
+        )
```

Comment:
Does this work? I think it might sometimes, but sometimes FITS images have a rotation in the WCS and then this comes out incorrectly?

Probably the right solution is to compute a footprint from the corners and to return it as a SkyPolygonRegion (or later to sub-sample the sides for better resolution). So call [WCS.calc_footprint](https://docs.astropy.org/en/stable/api/astropy.wcs.WCS.html#astropy.wcs.WCS.calc_footprint) and fill the points into a SkyPolygonRegion.

Suggest to leave at least a TODO comment to check that and add  a test for this later - or file a reminder follow-up issue referencing this comment.